### PR TITLE
Standardize nice_size() import.

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: lib/galaxy/model/__init__.py [hunk 1]
```diff
@@ -185,7 +186,7 @@ def get_disk_usage( self, nice_size=False ):
         if self.disk_usage is not None:
             rval = self.disk_usage
         if nice_size:
-            rval = galaxy.datatypes.data.nice_size( rval )
+            rval = galaxy.util.nice_size( rval )
```
Comment (bgruening) [inline lib/galaxy/model/__init__.py:189]: Any reason to not use: `from galaxy.util import nice_size` here?

Comment (nsoranzo) [inline lib/galaxy/model/__init__.py:189]: Yes, `nice_size` is also a parameter of this function (and of all the others below except one).

Comment (bgruening) [inline lib/galaxy/model/__init__.py:189]: Ok, thanks!

**Comment by bgruening** [comment]
:+1:

**Comment by dannon** [comment]
+1

**Pull request merged**